Permissions/ViewController.m

```diff
@@ -629,23 +628,17 @@ - (void) tableView:(UITableView *) tableView didSelectRowAtIndexPath:(NSIndexPat
 
 #pragma mark - Address Book
 
-- (ABAddressBookRef) addressBook {
+- (CNContactStore *) addressBook {
   return _addressBook;
 }
 
-- (void) setAddressBook:(ABAddressBookRef) newAddressBook {
+- (void) setAddressBook:(CNContactStore *) newAddressBook {
   if (_addressBook != newAddressBook) {
-    if (_addressBook != NULL) {
```

Comment:
Why do we removed these conditions?

Reply:
We removed this because there is no need to explicitly release and retain CNContactStore object.